docs/pywbemcli/commands.rst

```diff
@@ -2858,8 +2880,9 @@ The command line options for this command are:
 * ``--source-namespaces`` The name of WBEM namespaces on the server where
   indications originate.  Multiple namespaces may be defined as one or
   more namespaces per options instantiation where multiple namespaces are
-  separated by comma (","). For example, ``--source-namespaces
-  root/cimv2,root/cimv3"" or ``--source-namespaces root/cimv2 --source-namespaces root/cimv3``.
+  separated by comma (","). For example either, ``--source-namespaces
+  root/cimv2,root/cimv3"" or ``--source-namespaces root/cimv2 --source-namespaces root/cimv3``
```

Comment:
Change double quotes after cimv3 to backquotes.

Reply:
DONE